docs-chef-io/content/workstation/config_rb.md

```diff
@@ -49,6 +49,10 @@ This configuration file has the following settings:
   chef_server_url 'https://localhost/organizations/ORG_NAME'
   ```
 
+  {{< note >}}
+  If changes need to be made to any global end points like user or user keys, update `chef_server_url` in `knife.rb` to `https://localhost/` instead of `https://localhost/organizations/ORG_NAME`.
```

Comment:
We probably want to suggest the `--server-url` flag here as well so people don't think they need to edit their config.rb for these operations

Reply:
Done.